packages/local_auth/local_auth_ios/example/ios/RunnerTests/FLTLocalAuthPluginTests.m

```diff
@@ -269,4 +269,281 @@ - (void)testSkippedLocalizedFallbackTitle {
+  FLTLocalAuthPlugin *plugin = [[FLTLocalAuthPlugin alloc] init];
+  id mockAuthContext = OCMClassMock([LAContext class]);
+  plugin.authContextOverrides = @[ mockAuthContext ];
+
+  const LAPolicy policy = LAPolicyDeviceOwnerAuthenticationWithBiometrics;
+  OCMStub([mockAuthContext canEvaluatePolicy:policy error:[OCMArg setTo:nil]]).andReturn(YES);
+
+  FlutterMethodCall *call = [FlutterMethodCall methodCallWithMethodName:@"getEnrolledBiometrics"
+                                                              arguments:@{}];
+  XCTestExpectation *expectation = [self expectationWithDescription:@"Result is called"];
+  [plugin handleMethodCall:call
+                    result:^(id _Nullable result) {
+                      XCTAssertTrue([NSThread isMainThread]);
+                      XCTAssertTrue([result isKindOfClass:[NSArray class]]);
+                      XCTAssertEqual([result count], 1);
+                      XCTAssertEqualObjects(result[0], @"fingerprint");
+                      [expectation fulfill];
+                    }];
+
+  [self waitForExpectationsWithTimeout:kTimeout handler:nil];
+}
+
+- (void)testGetEnrolledBiometrics_withoutEnrolledHardware_iOS11 {
+  if (@available(iOS 11.0.1, *)) {
+    FLTLocalAuthPlugin *plugin = [[FLTLocalAuthPlugin alloc] init];
+    id mockAuthContext = OCMClassMock([LAContext class]);
+    plugin.authContextOverrides = @[ mockAuthContext ];
+
+    const LAPolicy policy = LAPolicyDeviceOwnerAuthenticationWithBiometrics;
+    void (^canEvaluatePolicyHandler)(NSInvocation *) = ^(NSInvocation *invocation) {
+      // Write error
+      NSError *__autoreleasing *authError;
+      [invocation getArgument:&authError atIndex:3];
+      *authError = [NSError errorWithDomain:@"error" code:LAErrorBiometryNotEnrolled userInfo:nil];
+      // Write return value
+      BOOL returnValue = NO;
+      NSValue *nsReturnValue = [NSValue valueWithBytes:&returnValue objCType:@encode(BOOL)];
+      [invocation setReturnValue:&nsReturnValue];
+    };
+    OCMStub([mockAuthContext canEvaluatePolicy:policy
+                                         error:(NSError * __autoreleasing *)[OCMArg anyPointer]])
+        .andDo(canEvaluatePolicyHandler);
+
+    FlutterMethodCall *call = [FlutterMethodCall methodCallWithMethodName:@"getEnrolledBiometrics"
+                                                                arguments:@{}];
+    XCTestExpectation *expectation = [self expectationWithDescription:@"Result is called"];
+    [plugin handleMethodCall:call
+                      result:^(id _Nullable result) {
+                        XCTAssertTrue([NSThread isMainThread]);
+                        XCTAssertTrue([result isKindOfClass:[NSArray class]]);
+                        XCTAssertEqual([result count], 0);
+                        [expectation fulfill];
+                      }];
+
+    [self waitForExpectationsWithTimeout:kTimeout handler:nil];
+  }
+}
+
+- (void)testGetEnrolledBiometrics_withoutEnrolledTouchID_preIOS11 {
```

Comment:
You can eliminate this test (which should fix the compile error), per my other comment about using `else` on the actual check.